controllers/front/OrderOpcController.php

```diff
@@ -95,10 +95,21 @@ private function checkWhetherAllConditionsAreApproved()
         return true;
     }
 
+    public function getPaymentOptionsAction()
+    {
+        return $this->renderPaymentOptions();
+    }
+
     public function init()
     {
         parent::init();
 
+        if (($action = Tools::getValue('action'))) {
+            $result = $this->{$action . 'Action'}();
```

Comment:
We had those checks in most of such a cases in PrestaShop that's why I'm asking :P